cinema_service/urls.py

```diff
@@ -4,5 +4,6 @@
 urlpatterns = [
     path("admin/", admin.site.urls),
     path("api/cinema/", include("cinema.urls", namespace="cinema")),
+    path("api/user/", include("user.urls", namespace="user")),
```

Comment:
Ensure that the `debug_toolbar` is only included in development environments. In production, this should be removed or conditionally included based on the `DEBUG` setting.